File: src/test/java/selenium/tests/AdminPage.java
package selenium.tests;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

/**
 * Created by Administrator on 2016-09-23.
 */
public class AdminPage extends Abstract {
    public AdminPage(WebDriver driver) {
        super(driver);
    }

    public AdddingPage goToAddingPage() {
        driver.findElement(By.xpath("//div[@class='wp-menu-image dashicons-before dashicons-admin-post']")).click();
        driver.findElement(By.xpath("//div[@class='wrap']//a[text()='Dodaj nowy']")).click();
        return new AdddingPage(driver);
    }

}

File: TestAutomation/tests/AdminPage.cs
using System;
using OpenQA.Selenium;

namespace TestAutomation.tests
{
    internal class AdminPage : Page
    {
        public AdminPage(IWebDriver driver) : base(driver)
        {
        }

        internal AddPostPage goToAddNewPost()
        {
            click(By.XPath("//*[@id='menu-posts']/a/div[contains(text(),'Wpisy')]"));
            click(By.XPath("//li[@id='menu-posts']//a[contains(text(),'Dodaj nowy')]"));
            return new AddPostPage(driver);
        }
    }
}